crates/figma_import/src/extended_layout_schema.rs

```diff
@@ -45,6 +45,20 @@ pub struct SpanData {
     pub max_span: bool,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct GridNodeSize {
+    /// The node name of an item that goes into a grid layout
+    #[serde(default)]
+    pub node_name: String,
+    /// Width of the node in Figma
```

Comment:
I think we should say "Width of the node in the design source" or something like that (and not mention Figma at all in the toolkit_schema side of the house).